.goreleaser.yml

```
@@ -8,9 +8,14 @@ builds:
       - linux
```

It looks like Terraform also recommends to build for `freebsd`. Let's add it:
```suggestion
      - freebsd
      - linux
```

---

Nice catch. Thanks :+1: